autoload/neomake/makers/ft/python.vim

```diff
@@ -251,9 +250,12 @@ endfunction
 " --fast-parser: adds experimental support for async/await syntax
 " --silent-imports: replaced by --ignore-missing-imports --follow-imports=skip
 function! neomake#makers#ft#python#mypy() abort
-    let defaults = ['--ignore-missing-imports', '--follow-imports=skip']
+    let args = ['--ignore-missing-imports', '--follow-imports=skip']
+    if neomake#utils#IsRunningWindows()
```

Comment:
@blueyed apologies but I've just come back to this and I realised there was a mistake in your refactor.

We only wanted to add `--fast-parser` for non-Windows user,  
where as your refactor looks to add it if you are a Windows user